ibm_catalog.json

```diff
@@ -258,12 +258,14 @@
 							"key": "en_email_list",
 							"type": "array",
 							"default_value": [],
+							"description": "The list of email addresses to notify when Security and Compliance Center triggers an event.",
```

Comment:
Its not just SCC events, its secrets manager too. Use the same description as core security services does